src/Module.Server/ModuleData/dtv_data.xml

```diff
@@ -56,23 +58,25 @@
     </Wave>
     <Wave>
       <Group classDivisionId="crpg_dtv_fancy_lad" count="1" />
-      <Group classDivisionId="crpg_dtv_fancy_noble" count="1" />
+      <Group classDivisionId="crpg_dtv_fancy_noble" count="0.9" />
+      <Group classDivisionId="crpg_dtv_fancy_cavalry" count="0.2" />
     </Wave>
     <Wave>
       <Group classDivisionId="crpg_dtv_fancy_lad" count="1" />
       <Group classDivisionId="crpg_dtv_fancy_noble" count="1" />
+      <Group classDivisionId="crpg_dtv_fancy_cavalry" count="0.3" />
```

Comment:
The fancy people are the worst IMO :D so I would decrease the infantry number

Reply:
done